bindings/python/htcondor2/_job_event.py

```diff
@@ -1 +1,62 @@
+from collections.abc import Mapping
+
+from ._common_imports import (
+    classad,
+)
+
+from ._job_event_type import JobEventType
+
+class JobEvent(Mapping):
+
+    def __init__(self, data : classad.ClassAd, event_text : str):
+        self._data = data
+        self._event_text = event_text
+
+
+    @property
+    def type(self) -> JobEventType:
+        return JobEventType(self._data["type"])
+
+
+    @property
+    def cluster(self) -> int:
+        return self._data["cluster"]
+
+
+    @property
+    def proc(self) -> int:
+        # The event ClassAd doesn't contain a Proc attribute if the original
+        # event's ProcID was less than or equal to 0.  ULogEvent's default
```

Comment:
```suggestion
        # event's ProcID was less than 0.  ULogEvent's default
```
A ProcID of 0 in the event does result in 'Proc' being set in the ClassAd form, from my reading of the code.

Reply:
Good catch.